### PR TITLE
Feature: implement download all

### PR DESCRIPTION
**Pull request by ikushum**
Implement download all data

### CODE REVIEW

File: modules/admin/src/views/ListDocuments2.vue [hunk 1]
```diff
@@ -126,18 +131,36 @@ export default {
...
+      let page = 1
+
+      while (this.count > rows.length) {
+        const params = {
+          page: page,
+          limit: 500,
+          orderby: '{"meta.created":-1}',
+        }
+
+        const response = (await request({ url: `/${this.collectionName}/`, params })).data
```
Comment (kane-mason) [inline modules/admin/src/views/ListDocuments2.vue:157]: please include the search query in this request

Comment (ikushum) [reply modules/admin/src/views/ListDocuments2.vue:157]: @kane-mason  But this request is supposed to fetch all the data for **Download All** button. Wouldn't it be in-accurate to filter data in this case?

Comment (kane-mason) [reply modules/admin/src/views/ListDocuments2.vue:157]: the way i see it, is downloading all the pages of the current view - so its somewhat subjective, i need it to be like this so this is the way we go for now :)

Comment (kane-mason) [reply modules/admin/src/views/ListDocuments2.vue:157]: also this.count is calculated based on the filtered results, so not doing it this way will probably truncate the data when a filter is present

Comment (ikushum) [reply modules/admin/src/views/ListDocuments2.vue:157]: Ah, I see. So I will include the search filters as well and everything should be fine.

**Pull request merged**